src/trusted/platform_qualify/arch/x86/vcpuid.c

```diff
@@ -386,13 +386,13 @@ static int asm_HasCX8(void) {
 #endif  /* 0 */
 #endif  /* 64-bit Windows */
 
-#if NACL_WINDOWS && (NACL_BUILD_SUBARCH == 64)
+#if !(NACL_WINDOWS && (NACL_BUILD_SUBARCH == 64)) || !defined(MSC_VER)
```

Comment:
This here inverts the condition in a way that doesn't look intentional and doesn't match the comment below.